modules/checking-cluster-resource-availability-and-utilisation.adoc

```diff
@@ -0,0 +1,41 @@
+// Module included in the following assemblies:
+//
+// *installing/validating-an-installation.adoc
+
+[id="checking-cluster-resource-availability-and-utilisation_{context}"]
+= Checking cluster resource availability and utilisation
```

Comment:
s/utilisation/utilization/

Reply:
Thank you. I have now resolved this in the title and in the module name.